oracle/oracle.go

```diff
@@ -271,18 +271,15 @@ func (o *OracleImpl) setLastSyncTime(t time.Time) {
 func (o *OracleImpl) GetPrices() types.TickerPrices {
 	prices := o.priceAggregator.GetAggregatedData()
 
-	// set metrics in background
-	go func() {
-		for cp, price := range prices {
-			floatValue, _ := price.Float64() // we ignore the accuracy in this conversion
-
-			o.metrics.UpdateAggregatePrice(
-				strings.ToLower(cp.String()),
-				cp.Decimals,
-				floatValue,
-			)
-		}
-	}()
+	for cp, price := range prices {
+		floatValue, _ := price.Float64() // we ignore the accuracy in this conversion
+
+		o.metrics.UpdateAggregatePrice(
+			strings.ToLower(cp.String()),
+			cp.Decimals,
+			floatValue,
+		)
+	}
```

Comment:
do we want to eventually flag this behavior?  I feel like in certain prod scenarios ppl might wanna not run this.

Reply:
this is a no op if metrics is disabled, what type of flagging did you have in mind?